lib/Differentiator/ReverseModeVisitor.cpp

```diff
@@ -2075,21 +2077,23 @@ Expr* ReverseModeVisitor::getStdInitListSizeExpr(const Expr* E) {
     // calls.
     llvm::SmallVector<Stmt*, 16> PostCallStmts{};
     if (pullbackFD) {
```

Comment:
```suggestion
    if (pullbackFD && !utils::hasEmptyBody(pullbackFD)) {
```

Can this become something like that?

Reply:
Then, it will attempt to use numerical diff